tephra-core/src/main/java/co/cask/tephra/snapshot/DefaultSnapshotCodec.java

```diff
@@ -38,8 +38,9 @@
 
 /**
  * Handles serialization/deserialization of a {@link co.cask.tephra.persist.TransactionSnapshot}
- * and its elements to {@code byte[]}.
+ * and its elements to {@code byte[]}. This codec is now deprecated and is replaced by {@link SnapshotCodecV2}.
  */
+@Deprecated
 public class DefaultSnapshotCodec implements SnapshotCodec {
```

Comment:
I didn't want to change existing class name since it is part of configuration value `data.tx.snapshot.codecs`